### PR TITLE
Bugfix: Fix broken headers in mdbook

### PR DESCRIPTION
**Pull request by Javagedes**
A previous commit to satisfy markdownlint requirements broke the headers in the mdbook. The `#` header shows up in the mdbook as a new section outside of chapters. Changing to `##` ended up breaking the intended formatting of the mdbook as those headers no longer showed up.

| Broken | Intended |
|--------|----------|
| image | image |

### CODE REVIEW

File: .github/workflows/ci-workflow.yml
Comment (makubacki) [inline .github/workflows/ci-workflow.yml:1]: These GitHub ref changes are ultimately coming in through another PR, right?

Comment (Javagedes) [reply .github/workflows/ci-workflow.yml:1]: Yes it was just testing. I'll drop the commit

Comment (Javagedes) [reply .github/workflows/ci-workflow.yml:1]: I'm still hoping we don't need those changes.

Comment (makubacki) [reply .github/workflows/ci-workflow.yml:1]: Me too :)

Comment (Javagedes) [reply .github/workflows/ci-workflow.yml:1]: Dropped!

**Comment by Javagedes** [comment]
close in favor of #104

**Pull request closed without merging**